multiterm/src/notebook.vala

```diff
@@ -149,7 +153,11 @@ namespace MultiTerm
 				context_menu.new_window_activate.connect(on_new_window_activate);
 				context_menu.move_to_location_activate.connect(on_move_to_location);
 			}
+#if MULTITERM_GTK3
+			context_menu.popup_at_pointer(event);
```

Comment:
This requires GTK+ 3.22 but I think the build system only checks for GTK+ >= 3.0, so if someone built it with any version of GTK+ between 3.0 and 3.22, it would cause a build error.

Reply:
Can I use normal gtk C-style version checks in valac?

Reply:
Not for compile-time versions (ie. `GTK_CHECK_VERSION` macros), Vala's preprocessor is too wimpy. You can pass along a macro from the build system using `-D` option if it's really needed.

Reply:
Ok, I tried defining something like ```MULTITERM_GTK_3_22``` but failed at changing the makefiles properly. So I leave this as is for now.